sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/models/BlobSetHttpHeadersHeaders.java

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.storage.blob.models;
+
+import com.azure.android.core.util.DateTimeRfc1123;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+import org.threeten.bp.OffsetDateTime;
+
+/**
+ * Defines headers for SetHTTPHeaders operation.
+ */
+@JacksonXmlRootElement(localName = "Blob-SetHttpHeaders-Headers")
+public final class BlobSetHttpHeadersHeaders {
+    /*
```

Comment:
I mentioned this on [another PR](https://github.com/Azure/azure-sdk-for-android/pull/386), but I believe that if we add these Javadoc explanations at the property level we can just reference them in the getters and setters.
```suggestion
    /**
```

Reply:
Since it sounds like there's a generator coming soon-ish, would you mind if we leave this for now?

Reply:
Sounds good to me.